### PR TITLE
Feature/#tra boot camp 50 

### PR DESCRIPTION
**Pull request by iamabdh**
https://trello.com/c/0epJcdS9

### CODE REVIEW

File: src/main/java/io/code/CodeExample.java [hunk 1]
```diff
@@ -9,4 +12,27 @@ public static void main(String[] arg) {
 	public int DummyFunction(int a) {
 		return a;
 	}
+
```
Comment (AnwaarAlbusaidi) [inline src/main/java/io/code/CodeExample.java:15]: Add Docs

**Review by AnwaarAlbusaidi — APPROVED** [review]
well Done

**Review by CodelineAtyab — APPROVED** [review]
Great

File: src/main/java/io/code/CodeExample.java (hunk 1 above)
Comment (iamabdh) [reply src/main/java/io/code/CodeExample.java:15]: noted with thanks

**Review by AmeerAllawati — APPROVED** [review]

**Review by BaraahAlKalbani — APPROVED** [review]

**Pull request merged**